src/Cloud/weglide/WeGlideObjects.hpp

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#include "util/StaticString.hxx"
+#include "time/BrokenDate.hpp"
+
+#include <cinttypes>
+#include <tchar.h>
+
+
+namespace WeGlide {
+
+struct User {
+  uint32_t id;
+  BrokenDate birthdate;
+  StaticString<0x80> name;
+
+  constexpr bool IsValid() const noexcept
+  {
+    return id > 0;
+  }
+
+  void Clear() noexcept
+  {
+    id = 0;
+    name.clear();
+    birthdate.Clear();
+  }
+};
+// because it is used in Settings and there it has to be 'trivial' type!
```

Comment:
What does that mean?

Reply:
The structure is usable only if it is a 'trivial type' - if I don't pay attention to this, I get an error somewhere else, which in itself is not self-explanatory - at least not for me. With the static_assertion in the following line, such an incorrect member type or member function is immediately displayed correctly.